free-1.md

```diff
@@ -5,47 +5,63 @@
+
 ### Why a language?
 
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+How many times a day to you say hey thats interesting or hey I need to remember this for later or you struggle to remember where you stored a password/peice of info. This language would allow the user to instanly store whatever they needed into a data base through a hopfully simple language on their phone or computer. For instace userA wants to remeber where they stored a book in their book collection of 100+ books. UserA could use the language by simply typing the name of the book/author to recover the infomation that is stored about the book in the data base. Another example UserB is a teacher and wants to keep track of a particular website for their students to use in years to come. Inseated of putting the website in a google app of some kind she can use the language to keep track of it the database.
+
 ### Why you?
 
 _What excites you about this idea? How did you come up with it?_
 
+What excites me about this language is its ability to communicate with data bases and its ability to adapt change and build on top of a data base. I feel like data bases are a growing feild and having a langauge that allows for quick and affective storage could be very versitile in the industry. I came up with this idea after our interveiw with one of the professors at Mudd and have continued to think about its possibilites with regards to my own expirence in the industry. 
+
 ### Domain
 
 _Describe the project's domain in five words._
 
+accessable langauge for quick/affective Information storage
+
 ### Interface (syntax)
 
 _How might the user interact with the language? What does programming look
 like? Why is this the right way to interact with the problem domain?_
 
+My though would be that the language would be a combination of a natural language and an optional coding like langauge. For instance users that want to use this aplication for personal knowlegde such as books, reports, websites ect... would have thia ablity to use either a phone or computer to efforlessly say something like " store book "title of book" "anything else they deem important about the book such as location in collection" " where the language would use precise key words to do an action. Additionally, mayble this language presents itself more useful for database programers that have to phisicallycode the database with the type of information to be stored, how much space to use, or other data base process that have to be completed each time you build a database. In this example we could have a DSL that still works roughly the same as above but may allow for more coding ability such as add, recover, delete, and edit funcftions. This would allow for programers to get rid of most of their set up and hard coding of the data base and move into just the what do I want to do with it phase. 
```

Comment:
I have some questions about operation, which you might address below, but in the world where it just "works" that would be really cool, and sounds very intuitive.